bottles/frontend/ui/about.ui

```diff
@@ -20,6 +20,7 @@ BrainBlasted https://github.com/BrainBlasted
 Francesco Masala &lt;mail@francescomasala.me&gt;
 Hari Rana (TheEvilSkeleton) https://theevilskeleton.gitlab.io/
 axtlos https://axtloss.github.io/</property>
+Oro https://github.com/orowith2os
```

Comment:
```suggestion
axtlos https://axtloss.github.io/
Oro https://github.com/orowith2os</property>
```

Reply:
Hm, but there are other people below that with no `</property>` tag.

Reply:
The other people are in a different property, https://github.com/bottlesdevs/Bottles/blob/79d03062701d52847941fae86453ac28899b6dd4/bottles/frontend/ui/about.ui#L24
The way you did it you'd be in no property and it should fail to build properly

Reply:
Ah, alright then. That makes sense.